python/ray/data/preprocessor.py

```diff
@@ -58,6 +59,8 @@ class FitStatus(str, Enum):
 
     # Preprocessors that do not need to be fitted must override this.
     _is_fittable = True
+    # Default batch format unless numpy is explicitly specified.
+    batch_format = "pandas"
```

Comment:
How is this used for the built in preprocessors? Do all preprocessors need to take in a `batch_format` arg?

Reply:
Also how would this work for Chain preprocessor?

Reply:
Yep we want to make `BatchMapper` args consistent with `map_batches` so that people can pass in a `batch_format`. Hmm then maybe we only want batch_format in `BatchMapper` level, not default to all preprocessors.

Reply:
Right I agree for BatchMapper! But not sure about the other preprocessors...let me think about it more